src/Stratis.Bitcoin.Features.SignalR/Controllers/SignalRController.cs

```diff
@@ -0,0 +1,52 @@
+namespace Stratis.Bitcoin.Features.SignalR.Controllers
+{
+    using System;
```

Comment:
`using`s should be at the top of the file.

Reply:
Done